test/y2network/widgets/interfaces_table_test.rb

```diff
@@ -133,5 +133,18 @@
         subject.handle
       end
     end
+
+    context "when the device is named by user" do
```

Comment:
I would expect this context as part of https://github.com/yast/yast-network/blob/77c7669cf136fdccffb2443354d999208431111b/test/y2network/widgets/interfaces_table_test.rb#L121

Reply:
```ruby
context "when the device is configured" do 
  context "and it is named by the user" do
```